src/languages/en.ts

```diff
@@ -628,7 +630,8 @@ export default {
         canceled: 'Canceled',
         posted: 'Posted',
         deleteReceipt: 'Delete receipt',
-        routePending: 'Route pending...',
+        routePending: 'Pending...',
```

Comment:
NAB: Since we also use it for distance let's just call it `pending`

Reply:
@neil-marcellini we already have "pending" translation but without "...". Should I rename it or concatenate translation string with "..." (it's used in several places)

Reply:
I like the key name `fieldPending` that you chose